Philips.CodeAnalysis.MaintainabilityAnalyzers/Philips.CodeAnalysis.MaintainabilityAnalyzers.md

```diff
@@ -66,7 +66,7 @@
 | PH2113  | Merge If Statements                          | Nested If statements lacking else clauses and containing the same body can be safely merged to reduce cognitive load |
 | PH2114  | Avoid empty statement                        | Avoid empty statements. |
 | PH2115  | Every Lambda expression on separate line     | Avoid putting multiple lambda statements on a single line for readability. |
-| PH2116  | Avoid ArrayList                              | Usage of Arraylist is discouraged by Microsoft for performance reasons, use List<T> instead. |
+| [PH2116](../Documentation/Diagnostics/PH2116.md)  | Avoid ArrayList                              | Usage of Arraylist is discouraged by Microsoft for performance reasons, use List<T> instead. |
```

Comment:
Typically, repo authors use "docs" rather than "Documentation"

Reply:
True, because of GH-Pages